### PR TITLE
chore(MySQL): Deprecate 5.6

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Deprecate 5.6

## How to Test the Changes:

:shrug: Try using it with MySQL 5.7

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/common/Database/DatabaseWorkerPool.cpp [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #include "DatabaseWorkerPool.h"
 #include "DatabaseEnv.h"
 
-#define MIN_MYSQL_SERVER_VERSION 50600u
-#define MIN_MYSQL_CLIENT_VERSION 50600u
+#define MIN_MYSQL_SERVER_VERSION 50700u
+#define MIN_MYSQL_CLIENT_VERSION 50700u
```
Comment (Kitzunu) [inline src/common/Database/DatabaseWorkerPool.cpp:11]: I don't know if this is correct, @Yehonal perhaps you know?

Comment (FrancescoBorzi) [reply src/common/Database/DatabaseWorkerPool.cpp:11]: @Kitzunu have you tested this PR with MySQL 5.7 ?

Comment (Kitzunu) [reply src/common/Database/DatabaseWorkerPool.cpp:11]: No i dont have it

Comment (Kitzunu) [reply src/common/Database/DatabaseWorkerPool.cpp:11]: I have only tested 8.0 which works, but someone that has 5.7 downloaded can test

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by pangolp** [comment]
Attached are images of the tests, I hope they are useful.

version
cmake
visual

**Review by pangolp — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
excellent @pangolp thanks for testing this. Good job @Kitzunu of course!

**Pull request merged**